### ISSUE TITLE
flake8 does not report D107 when disabling noqas

### ISSUE THREAD

**Issue by asottile** [PyCQA/flake8#788]
In GitLab by @pawamoy on Mar 14, 2021, 10:45

*Please describe how you installed Flake8*

As a Poetry dependency. It's equivalent to

```
$ pip install flake8
```

*Please provide the exact, unmodified output of `flake8 --bug-report`*

<details><summary>Flake8 bug report</summary>

```json
{
  "dependencies": [],
  "platform": {
    "python_implementation": "CPython",
    "python_version": "3.7.5",
    "system": "Linux"
  },
  "plugins": [
    {
      "is_local": false,
      "plugin": "black",
      "version": "0.2.1"
    },
    {
      "is_local": false,
      "plugin": "flake8-bandit",
      "version": "2.1.2"
    },
    {
      "is_local": false,
      "plugin": "flake8-bugbear",
      "version": "20.11.1"
    },
    {
      "is_local": false,
      "plugin": "flake8-comprehensions",
      "version": "3.3.1"
    },
    {
      "is_local": false,
      "plugin": "flake8-darglint",
      "version": "1.7.0"
    },
    {
      "is_local": false,
      "plugin": "flake8-docstrings",
      "version": "1.5.0, pydocstyle: 5.1.1"
    },
    {
      "is_local": false,
      "plugin": "flake8-pytest-style",
      "version": "1.4.0"
    },
    {
      "is_local": false,
      "plugin": "flake8-string-format",
      "version": "0.3.0"
    },
    {
      "is_local": false,
      "plugin": "flake8-tidy-imports",
      "version": "4.2.1"
    },
    {
      "is_local": false,
      "plugin": "flake8-variables-names",
      "version": "0.0.4"
    },
    {
      "is_local": false,
      "plugin": "flake8_builtins",
      "version": "1.5.2"
    },
    {
      "is_local": false,
      "plugin": "mccabe",
      "version": "0.6.1"
    },
    {
      "is_local": false,
      "plugin": "naming",
      "version": "0.11.1"
    },
    {
      "is_local": false,
      "plugin": "pycodestyle",
      "version": "2.6.0"
    },
    {
      "is_local": false,
      "plugin": "pyflakes",
      "version": "2.2.0"
    },
    {
      "is_local": false,
      "plugin": "pylint",
      "version": "0.0.0"
    },
    {
      "is_local": false,
      "plugin": "wps-light",
      "version": "0.15.2"
    }
  ],
  "version": "3.8.4"
}
```

</details>

*If this is a bug report, please explain with examples (and example code) what you expected to happen and what actually happened.*

Thanks to flake8-docstrings (and pydocstyle), flake8 reports [D107 violations (missing docstring in `__init__`)](https://github.com/PyCQA/pydocstyle/blob/master/src/pydocstyle/violations.py#L221) on some lines. When I `noqa` them, flake8 stops reporting them. But when I pass `--disable-noqa`, flake8 does not report them.

Setup:

```
git clone https://github.com/pawamoy/repro-flake8
cd repro-flake8
python3 -m venv venv
. venv/bin/activate
pip install flake8 flake8-docstrings
```

Reproduce:

```console
$ flake8 repro.py 
repro.py:7:1: D107 Missing docstring in __init__
repro.py:10:1: D105 Missing docstring in magic method
repro.py:17:1: D107 Missing docstring in __init__

$ flake8 repro_with_noqa.py
repro_with_noqa.py:10:1: D105 Missing docstring in magic method

$ flake8 repro_with_noqa.py --disable-noqa
repro_with_noqa.py:10:1: D105 Missing docstring in magic method
```

**Comment by asottile** [comment]
In GitLab by @asottile on Mar 14, 2021, 10:47

wrong project, this will be fixed by https://gitlab.com/pycqa/flake8-docstrings/-/merge_requests/29

**Comment by asottile** [comment]
In GitLab by @pawamoy on Mar 14, 2021, 11:01

I wouldn't have found this myself.

Glad to see it's already being fixed, thanks!
